### ISSUE TITLE
A11y_AzuredataStudio_ManageAccessForFile_ScreenReader:Voice Over is Not announcing the text associated with Label.

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7942]
GitHubTags: #A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG1.3.1;#WCAG4.2.5;#DesktopApp;#Mac;#Voice Over;
**Environment Details: **
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit:6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: MacOS Mojave 10.14.6
Additional Details:
Tool Name : VoiceOver
Steps to Reproduce:
Install and Launch the application the azure data studio application.
2.Connet to the servers with the required credentials.
3.Navigate to the control panel and click on the “Jupyter books” icon.
4.Navigate till HDFS folder and right click on it and click on manage access.
5.Start Voice Over and observe the announcement.
Actual:
Voice Over is Not announcing the text associated with Label.
Expected:
Voice Over should announce the text associated with label.
User Impact:
Screen Reader user will not able to relate with the content.
Attachment for Reference:
[7942_A11y_AzureDataStudio_Manageaccesfile _ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3761464/7942_A11y_AzureDataStudio_Manageaccesfile._ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
@V-sheman I'm not sure what issue you're referring to here exactly. What do you mean the "text associated with label"? The screen reader seems to be reading the label text off just fine in your video and I've confirmed that it announces the labels correctly for me as well.

**Comment by Charles-Gagnon** [comment]
I'm closing this as there doesn't appear to be any problem here. Let me know if you can provide more information!
